tests/mentorship_relation/test_dao_accept_request.py

```diff
@@ -53,16 +53,31 @@ def test_dao_requester_tries_to_accept_mentorship_request(self):
 
         result = DAO.accept_request(self.first_user.id, self.mentorship_relation.id)
 
-        self.assertEqual((messages.CANT_ACCEPT_MENTOR_REQ_SENT_BY_USER, 400), result)
+        # If a user tries to accepts a request where he is not involved, then
```

Comment:
Changes in this file not required for now

Reply:
But without these changes the tests would fail, simply because the added error messages are not  tested in the DAO file